Makefile

```diff
@@ -79,32 +79,12 @@ $(PROTOC):
 # Integration test for sdk client
 .PHONY: test-sdk
 test-sdk: clean-sdk build-sdk
-	DUMMY_BIN=$(DUMMY_BIN) LOOKOUT_BIN=$(LOOKOUT_BIN) $(GOCMD) run sdk-test/main.go
+	DUMMY_BIN=$(DUMMY_BIN) LOOKOUT_BIN=$(LOOKOUT_BIN) $(GOCMD) run cmd/sdk-test/main.go
 
 # Integration test for lookout serve
 .PHONY: test-json
-test-json: build-sdk
-	$(DUMMY_BIN) serve &>/dev/null & \
-	DUMMY_PID=$$!; \
-	cat fixtures/events.jsonl | $(LOOKOUT_BIN) serve --provider json dummy-repo-url > serve.txt 2> serve-err.txt & \
```

Comment:
A bit confused, could you help me understand: if all the logic from here was ported to server-test - is that expected not to have any mentions of `events.jsonl` below?

Reply:
All the logic was ported.
Every single input from `events.jsonl` is run by it now (in the same way through stdin but with go instead of bash).
It gives much more fine-grained control on what to what/when to send and what is more important - output related only to failing test instead of the huge log of everything.
So I don't see any reason to pipe `events.jsonl` into lookout anymore.

Does it answer your question?

Reply:
Yes, makes perfect sense!

It's just that I can not find `events.jsonl` file neither mentioned/read, nor remove in this PR below.